core/app/models/spree/order/store_credit.rb

```diff
@@ -11,7 +11,7 @@ def covered_by_store_credit?
       def total_available_store_credit
         return 0.0 unless user
 
-        user.total_available_store_credit
+        user.total_available_store_credit(self.currency, store)
```

Comment:
```suggestion
        user.total_available_store_credit(self.currency, self.store)
```

if we're using `self.` convention, let's use is everywhere